m08-react

iamabhutta opened the pull request:
updated slides based on fa24 feedback
TODO: Not sure what to do for Mitch's long comment for these slides (https://piazza.com/class/lz8s9rzs1dj30h/post/309).
TODO: Need to review activity and see if we need to edit anything.

iamabhutta committed:
m08-react

Commit:
reworked slides about setters

mwand commented:
>updated slides based on fa24 feedback

Also rewrote s14.


>TODO: Not sure what to do for Mitch's long comment for these slides (https://piazza.com/class/lz8s9rzs1dj30h/post/309).

Shuffled slides s18 et seq to incorporate some of this feedback. 


>TODO: Need to review activity and see if we need to edit anything.

- Activity runs as is.
- Need to update dependencies, but not urgent (give to a TA?)
- Is the task description in the standard place for the "single source of truth"?

iamabhutta commented:
Updated title of S18 to "Components don't Change their State directly (or synchronously)". Also updated some font sizes.
Will assign it to a TA to check on dependencies for the activity and then will update.
Rest lgtm

iamabhutta committed:
changes font sizes, updated S18 title
